packages/manager/apps/web/client/app/domain/list/list-domain-layout.controller.js

```diff
@@ -94,6 +99,30 @@ export default class ListDomainLayoutCtrl extends ListLayoutHelper.ListLayoutCtr
     };
   }
 
+  getDate(domain, key) {
+    const date = new Date(domain[key]);
+    const locale = this.coreConfig.getUserLocale().replace('_', '-');
```

Comment:
@dectotam can you please use `ovhSubsidiary` instead of `userLocale` because an FR user can use `it_IT` as a language in Manager. So this can cause issues.
You can refer https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/dedicatedCloud-datacenter-add.controller.js#L99-L108 for a similar case